common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -139,6 +140,25 @@ object GettyParser extends ImageProcessor {
   }
 }
 
+object GettyCreditParser extends ImageProcessor {
+  import UsageRightsConfig.payGettySourceList
+
+  def apply(image: Image): Image = {
+    val isPayForGettySource = image.metadata.source.exists(payGettySourceList.contains)
+    val gettyCredits = List("getty images", "afp/getty images")
+    val isGettyCredit = image.metadata.credit.map(_.toLowerCase).exists(gettyCredits.contains)
```

Comment:
This is for images without the XMP data, so we wouldn't have supplier or the collection at this point.